docs/build-clio.md

```diff
@@ -88,6 +88,31 @@ If successful, `conan install` will find the required packages and `cmake` will
 > [!TIP]
 > To generate a Code Coverage report, include `-o coverage=True` in the `conan install` command above, along with `-o tests=True` to enable tests. After running the `cmake` commands, execute `make clio_tests-ccov`. The coverage report will be found at `clio_tests-llvm-cov/index.html`.
 
+### Generating API docs for Clio
+
+The API documentation for Clio is generated by [Doxygen](https://www.doxygen.nl/index.html). If you want to generate the API documentation when building Clio, make sure to install Doxygen on your system.
+
+To generate the API docs:
+
+1. First, include `-o docs=True` in the conan install command. For example:
+
+    ```sh
+    conan install .. --output-folder . --build missing --settings build_type=Release -o tests=True -o lint=False -o docs=True
+    ```
+
+2. Once that has completed successfully, run the `cmake` command and add the `--target docs` option:
+
+   ```sh
+   cmake -DCMAKE_TOOLCHAIN_FILE:FILEPATH=build/generators/conan_toolchain.cmake -DCMAKE_BUILD_TYPE=Release .. \ 
+   cmake --build . --parallel 8 --target docs
+   ```
+
+3. In the `docs` directory, you should see a folder called `html`. Go to the folder to view the generated files.
```

Comment:
The `html` folder with the doc is not inside `docs` but inside your `build` folder

Reply:
I just ran this (in Docker if it makes any difference) and that's where they ended up.

Reply:
Yep it's definitely creating this in `docs/html` folder when I run this. I can change this back to `build`, but just FYI it's not matching what I'm seeing Docker.

Reply:
You are correct, it's ending up in `build/docs/html`, i thought you were referring to our `docs` folder in the repo 😃

Reply:
Haha, ok I'll put it back in 🙌🏾